conans/client/cmd/uploader.py

```diff
@@ -339,151 +233,127 @@ def _compress_package_files(self, layout, pref):
                 CONANINFO: files[CONANINFO],
                 CONAN_MANIFEST: files[CONAN_MANIFEST]}
 
-    def _package_integrity_check(self, pref):
-        # If package has been modified remove tgz to regenerate it
-        self._output.rewrite_line("Checking package integrity...")
+    def _check_upstream_recipe(self, recipe, remote, force):
+        self._output.info("Checking which revisions exist in the remote server")
```

Comment:
**Minor detail:** when you upload multiple revisions for me it's a bit confusing seeing in the output this repeated several times, like this:

```
(conan-venv) ➜  upload conan upload hello/0.2 -r test-repo
Preparing artifacts to upload
Checking which revisions exist in the remote server
Checking which revisions exist in the remote server
Uploading artifacts
Uploading hello/0.2
Uploading hello/0.2#4053:8860#1cac
Uploading hello/0.2
Uploading hello/0.2#7e26:8860#bb86
Uploading hello/0.2#7e26:8860#3588
```
Maybe we can add which revision it is checking? Something like: 
`Checking if revision hello/0.2#4053:8860 exists in the remote server`

Reply:
Yes, the output is still to be revisited completely, I'd say to leave it out of the scope of this PR. I removed the progress bars, and left it as it was, it needs some love (all the output, not just this)